unity/unity-embed-host/CoreCLRHost.cs

```diff
@@ -168,6 +168,22 @@ public static int array_length([NativeCallbackType("MonoArray*")] IntPtr array)
         [NativeCallbackType("MonoMethod*")] IntPtr method,
         [NativeCallbackType("MonoClass*")] IntPtr refclass) =>
         MethodInfo.GetMethodFromHandle(method.MethodHandleFromHandleIntPtr(), RuntimeTypeHandle.FromIntPtr(refclass)).ToNativeRepresentation();
+    
+    [return: NativeCallbackType("void*")]
+    public static IntPtr unity_method_get_function_pointer([NativeCallbackType("MonoMethod*")] IntPtr method)
+    {
+        try
+        {
+
+            // TODO: Assert method is decorated with UnmanagedCallersOnly
```

Comment:
Should this TODO be addressed?

Reply:
I'll leave it for now. Ideally we'd only check in a debug/development build and not in release.